### PR TITLE
Increase epoch to 8 hours, increase block cost to 40M, and bls agg co…

### PR DESCRIPTION
**Pull request by mariano54**
…st to 1ms.

Please wait for tests to pass

### CODE REVIEW

**Review by hoffmang9 — APPROVED** [review]
+1

File: src/consensus/condition_costs.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 class ConditionCost(Enum):
     # Condition Costs
-    AGG_SIG = 20
+    AGG_SIG = 92  # 1 ms BLS verify = 10,000 clvm cost / 108 cost multiplier
```
Comment (arvidn) [inline src/consensus/condition_costs.py:6]: we don't enforce a max cost yet though, right?

File: src/consensus/default_constants.py [hunk 2]
```diff
@@ -53,9 +53,10 @@
     # 1 vByte = 108 clvm cost units
     "CLVM_COST_RATIO_CONSTANT": 108,
     # Max block cost in clvm cost units (MAX_BLOCK_COST * CLVM_COST_RATIO_CONSTANT)
-    "MAX_BLOCK_COST_CLVM": 23077872,
+    # "MAX_BLOCK_COST_CLVM": 23077872,
+    "MAX_BLOCK_COST_CLVM": 40000000,  # Based on arvid analysis
```
Comment (arvidn) [inline src/consensus/default_constants.py:57]: we don't use this yet, which I believe is fine until we have transactions

**Review by arvidn — APPROVED** [review]

File: src/consensus/condition_costs.py (hunk 1 above)
Comment (mariano54) [reply src/consensus/condition_costs.py:6]: We do in testnet

File: src/consensus/default_constants.py (hunk 2 above)
Comment (mariano54) [reply src/consensus/default_constants.py:57]: We do use it after transaction freeze (in testnet)

**Pull request merged**